components/application-broker/internal/broker/provision_test.go

```diff
@@ -558,9 +668,23 @@ func TestDoKnativeResourceProvision(t *testing.T) {
 	}
 
 	for _, tc := range []testCase{
+		{
+			name:                           "provision success with new eventing flow",
+			givenCanProvisionOutput:        access.CanProvisionOutput{Allowed: true},
+			newEventingFlow:                true,
+			expectedOpState:                internal.OperationStateSucceeded,
+			expectedOpDesc:                 "provisioning succeeded",
+			expectedEventActivationCreated: true,
+			expectedInstanceState:          internal.InstanceStateSucceeded,
+			initialObjs:                    []runtime.Object{},
+			expectCreates: []runtime.Object{
+				bt.NewIstioPeerAuthentication(string(appNs), fmt.Sprintf("%s%s", appNs, peerAuthNameSuffix)),
+			},
+		},
 		{
 			name:                           "provision fail namespace not found",
 			givenCanProvisionOutput:        access.CanProvisionOutput{Allowed: true},
+			newEventingFlow:                false,
```

Comment:
(nit) other test cases (with the new flow disabled) does not have this because it's implicit.

Reply:
Added to all of them to be explicit